scripts/orthomcl-setup-database.pl

```diff
@@ -16,15 +16,16 @@
 
 my $default_config = "$script_dir/../etc/orthomcl.config.example";
 
-my $usage = "Usage: ".basename($0)." --user [user] --password [password] --host [host] --database [database]\n".
+my $usage = "Usage: ".basename($0)." --user [user] --password [password] --host [host] --database [database] --outfile [outfile]\n".
```

Comment:
Also, we should maybe mention that this script only works for a MySQL database here.